tests/nlu/test_config.py

```diff
@@ -99,43 +101,78 @@ def test_default_config_file():
     assert len(final_config) > 1
 
 
-def test_set_attr_on_component(pretrained_embeddings_spacy_config):
-    idx_classifier = pretrained_embeddings_spacy_config.component_names.index(
-        "SklearnIntentClassifier"
-    )
-    idx_tokenizer = pretrained_embeddings_spacy_config.component_names.index(
-        "SpacyTokenizer"
+def test_set_attr_on_component():
```

Comment:
should we test set attribute on different component? since we plan to deprecate sklearn